### DISCUSSION TITLE
Diving Suits - Leak oxygen before full suit failure

### DISCUSSION THREAD

**Discussion by jjtParadox** [PeasantFoundry/Mechtrauma#66] [Ideas]
I made a personal modification to Mechtrauma's diving suits that I quite liked and wanted to share. I kept the suit degradation the same but added a "Rapidly losing oxygen" effect when the suit goes below 5% condition, as if you were by one of those green cave vents.

The end result is that using a low condition diving suit is still a bad idea, but there's a bit of padding between "this suit is fine to use" and "you'll instantly die if you try to use this suit".

Do you all think this would be a good addition to the mod?

**Comment by MapleWheels** [comment]
I personally like this idea very much. It also adds a consequential feel to the idea of something being "worn out" or on the verge of failure.

**Comment by MapleWheels** [comment]
On PR #69 .

**Comment by TBN-MapleWheels** [comment]
Implemented by @PeasantFoundry via manual recreation.
